gazebo_plugins/src/gazebo_ros_camera.cpp

```diff
@@ -119,6 +174,30 @@ void GazeboRosCamera::Load(gazebo::sensors::SensorPtr _sensor, sdf::ElementPtr _
   RCLCPP_INFO(impl_->ros_node_->get_logger(), "Publishing camera info to [%s]",
     impl_->camera_info_pub_->get_topic_name());
 
+  if (impl_->sensor_type_ == GazeboRosCameraPrivate::DEPTH) {
+    // Depth image publisher
+    impl_->depth_image_pub_ = image_transport::create_publisher(impl_->ros_node_.get(),
+        impl_->camera_name_ + "/image_depth");
```

Comment:
For rviz2, we will have to name it `depth/image_raw` and `depth/image_raw/camera_info` because rviz2 is expecting `camera_info` to be appended to the image_topic

Reply:
Also normal rgb camera should have `image_raw` and `image_raw/camera_info` then